server/etcdserver/server.go

```diff
@@ -2209,12 +2211,30 @@ func (s *EtcdServer) monitorKVHash() {
 		if !s.isLeader() {
 			continue
 		}
-		if err := monitor.periodicCheck(); err != nil {
+		if err := s.corruptionChecker.PeriodicCheck(); err != nil {
 			lg.Warn("failed to check hash KV", zap.Error(err))
 		}
 	}
 }
 
+func (s *EtcdServer) monitorCompactHash() {
+	if !s.Cfg.CompactHashCheckEnabled {
+		return
+	}
+	t := s.Cfg.CompactHashCheckTime
```

Comment:
Should we apply a default value if it's 0 or should we disable it if it's 0?

Reply:
Added a validation to prevent negative value.

Reply:
OK